codebase2/codebase-sqlite/package.yaml

```diff
@@ -2,12 +2,9 @@ name: unison-codebase-sqlite
 github: unisonweb/unison
 
 library:
-  when:
-    - condition: false
-      other-modules: Paths_unison_codebase_sqlite
```

Comment:
I think this might now result in spurious re-builds, but obviously we need this module to exist.

Reply:
I think it still exists even with these lines there